packages/trpc/server/routers/viewer/organizations/listOtherTeams.handler.ts

```diff
@@ -9,38 +9,20 @@ type ListOptions = {
 };
 
 export const listOtherTeamHandler = async ({ ctx }: ListOptions) => {
-  const teamsInOrgIamNotPartOf = await prisma.membership.findMany({
+  const teamsInOrgIamNotPartOf = await prisma.team.findMany({
     where: {
-      userId: {
-        not: ctx.user.id,
+      parent: {
+        id: ctx.user?.organization?.id,
```

Comment:
Here is the main issue I was looking first on memberships but when a team doesn't have memberships it wasn't listed.

Reply:
Shouldn't a team always have atleast it's creator as the member?